index.d.ts

```diff
@@ -2,10 +2,15 @@
 // Project: https://github.com/sudodoki/copy-to-clipboard
 // Definitions by: Denis Carriere <https://github.com/DenisCarriere>, MartynasZilinskas <https://github.com/MartynasZilinskas>
 
+interface StyleOption {
+  background?: string;
+  text?: string;
+}
+
 interface Options {
   debug?: boolean;
   message?: string;
-  format?: string; // MIME type
+  style?: boolean | StyleOption; // MIME type
 }
```

Comment:
I don't know about typescript, I didn't check this is working as expected, so please double check!